Cargo.toml

```diff
@@ -41,8 +41,8 @@ rustdoc-args = ["--document-private-items", "--generate-link-to-definition"]
 
 [dependencies]
 abscissa_core = { version = "0.7.0", default-features = false, features = ["application"] }
-rustic_backend = { version = "0.2.1", features = ["cli"] }
-rustic_core = { version = "0.3.1", features = ["cli"] }
+rustic_backend = { git = "https://github.com/rustic-rs/rustic_core.git", features = ["cli"] }
+rustic_core = { git = "https://github.com/rustic-rs/rustic_core.git", features = ["cli"] }
```

Comment:
I think we may want to release the changes first without re-introducing git dependencies again, changing them back later. Now that releasing should be just merging the release pr.

Reply:
If we wait for a release of rustic_core, we'll pile up PRs in rustic which depend on these things but cannot be merged. We won't have the features in the nightly builds and will not get any feedback until we release a rustic release.

As many changes in rustic are changes in rustic_core, I see no alternative than using a git dependency here.

Reply:
Ok, I understand. Do you think it would be viable to up the release cadence for `rustic_core` and say we do at least one release in a week with things that have been merged, even if small one's? It would be just a click away, we could release another `rustic_core` version today, for example.